moc_sprint_tools/sprintman.py

```diff
@@ -65,3 +70,51 @@ def create_sprint(self, name):
             board.create_column(column)
 
         return board
+
+    def copy_card(self, source_card, destination_column):
+        content = source_card.get_content()
+
+        if content:
+            if isinstance(content, github.Issue.Issue):
+                content_type = 'Issue'
+            elif isinstance(content, github.PullRequest.PullRequest):
+                content_type = 'PullRequest'
+            else:
+                LOG.warning("couldn't copy card %s with unkown type", source_card)
+                return
+
+            LOG.info('adding card "%s" to column %s',
+                     content.title,
+                     destination_column.name)
+            destination_column.create_card(
+                content_id=content.id,
+                content_type=content_type,
+            )
+        elif source_card.note:
+            destination_column.create_card(note=source_card.note)
+        else:
+            raise ValueError(f'card {source_card.id} has no content')
+
+    def copy_board(self, source_board, destination_board,
+                   columns=None, ignore_columns=None):
```

Comment:
we could just set `ignore_columns` to `defaults.default_skip_copy` right here, instead of setting it to `None` and then overriding it later.

Reply:
If you take a look at how this is called from the `copy-cards` command ([here](https://github.com/CCI-MOC/sprint-tools/pull/21/files#diff-e3578f51d9a80f7bcac61942964425aaf16230e662bb314386fbd87de0ee3836R41)), you'll note that it is called with `ignore_columns` set to the value of the `--ignore-columns` command line option...which will be `None` by default. By setting the default inside the function instead of in the parameter list, we are able to keep the default logic here in this function rather than pushing it out to the CLI methods.

Reply:
took me a while to figure out. 

so if we set it to `defaults.default_skip_copy` in parameter list here then the caller could override it and set it back to `None`

Reply:
Right.